src/Formidable.ml

```diff
@@ -366,10 +366,15 @@ module Make
       let children = props##children in
       let value = Optic.Lens.view props##lens values in
       let isFocused, setIsFocused = React.useState (fun () -> false) in
+
       let validationNames =
         Option.fold []  (List.flatMap (Validations.getName >> Relude.String.splitAsList ~delimiter:"<or>")) props##validations
       in
-
+      
+      let hasValidation name =
+        validationNames |> List.containsBy (==) name         
```

Comment:
Does `let hasValidation = List.containsBy (==) name` compile?

Reply:
`let hasValidation = List.containsBy (==) name` it doesnt complie  (unbound for name
I think it is a function here right so maybe we couldn't make by that maybe I am wrong?

Reply:
sorry, bad copy/paste: `let hasValidation = List.containsBy (==)`

Reply:
it doesnt work :(